ruby_03-professional_rails_applications/building-a-vps.markdown

```diff
@@ -308,7 +308,15 @@ them which application process to use when serving requests.
 ### Configuring NGINX
 
 First, we're going to need to know where to tell Passenger to look for an installed version of Ruby.
-Fortunately passenger includes a script for finding this information, which we can use like so:
+Fortunately passenger includes a script for finding this information. Prior to setting that up we'll need to know where our installation of Ruby is hiding:
+
+```sh
+which ruby
+```
+
+You should get something like `/home/deploy/.rvm/rubies/ruby-2.2.1/bin/ruby`
+
+And then we can configure the passenger script like so:
```

Comment:
i dont like the wording of this b/c we're not actually "configuring the passenger script" -- just seeing where it is.